tools/cbmc/proofs/Makefile.template

```diff
@@ -110,11 +113,10 @@ $(ENTRY).goto: $(ENTRY)5.goto
 # Rules for running CBMC
 
 goto:
-	$(MAKE) patch
-	$(MAKE) $(ENTRY).goto
+	$(MAKE) -B $(ENTRY).goto
 
 cbmc.txt: $(ENTRY).goto
-	- cbmc $(CBMCFLAGS) --unwinding-assertions --trace @RULE_INPUT@ > $@ 2>&1
```

Comment:
In the run script, you'll see that if this invocation of CBMC returns an error, then we indicate a proof failure.

Unfortunately `make` returns `2` if the command failed, and it disregards the return code. So we won't be able to distinguish between tool and proof failures. The way to eventually accomplish this is to run CBMC directly from litani rather than doing `make cbmc`, so that litani can distinguish between return codes of `0`, `10`, and others, but I'll leave that for the future.

Reply:
Nonetheless, this means running "make report" locally when there is a verification failure will fail to generate the report.